user_docs/en/changes.md

```diff
@@ -104,6 +104,9 @@ Add-ons will need to be re-tested and have their manifest updated.
 * Added the [VS Code workspace configuration for NVDA](https://nvaccess.org/nvaccess/vscode-nvda) as a git submodule. (#17003)
 * In the `brailleTables` module, a `getDefaultTableForCurrentLang` function has been added (#17222, @nvdaes)
 * Retrieving the `labeledBy` property now works for UIA elements supporting the corresponding `LabeledBy` UIA property. (#17442, @michaelweghorn)
+* Added the following extension points (#17428):
```

Comment:
This is optional, but maybe you are interested in adding your GitHub contact ID:

```suggestion
* Added the following extension points (#17428, @ctoth):
```